### PR TITLE
test(rebrand): update rebranded component tests

### PR DESCRIPTION
**Pull request by anuraghazra**
- Web ✅ 
- Native ✅

### CODE REVIEW

File: packages/blade/rebranded-components.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// baseBoxStyles
+const rebrandedComponents = [
```
Comment (anuraghazra) [inline packages/blade/rebranded-components.js:2]: @saurabhdaware @kamleshchandnani let me know If i missed any related/Base or other components.

File: packages/blade/.storybook/react/preview-head.html [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 * :where(p:not(.sb-anchor, .sb-unstyled, .sb-unstyled p)),
-* :where(h1:not(.sb-anchor, .sb-unstyled, .sb-unstyled h1)),
-* :where(h2:not(.sb-anchor, .sb-unstyled, .sb-unstyled h2)),
-* :where(h3:not(.sb-anchor, .sb-unstyled, .sb-unstyled h3)),
-* :where(h4:not(.sb-anchor, .sb-unstyled, .sb-unstyled h4)),
 * :where(ul:not(.sb-anchor, .sb-unstyled, .sb-unstyled ul)),
 * :where(ol:not(.sb-anchor, .sb-unstyled, .sb-unstyled ol)),
 * :where(li:not(.sb-anchor, .sb-unstyled, .sb-unstyled li)) {
-    font-family: Lato,-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Helvetica,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol" !important;
+    font-family: "Inter", -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif !important;
```
Comment (chaitanyadeorukhkar) [inline packages/blade/.storybook/react/preview-head.html:7]: What have you changed here?

File: packages/blade/jest.native.config.js [hunk 3]
```diff
@@ -13,7 +15,7 @@ module.exports = {
     },
   },
   moduleFileExtensions: ['native.ts', 'native.tsx', 'ts', 'tsx', 'js', 'json', 'node'],
-  testMatch: ['**/*.test.{ts,tsx}'],
+  testMatch: rebrandedComponents.map((c) => `**/${c}.native.test.{ts,tsx}`),
```
Comment (chaitanyadeorukhkar) [inline packages/blade/jest.native.config.js:18]:
```suggestion
  testMatch: rebrandedComponents.map((component) => `**/${component}.native.test.{ts,tsx}`),
```

File: packages/blade/jest.native.config.js [hunk 4]
```diff
@@ -1,9 +1,11 @@
+const rebrandedComponents = require('./rebranded-components');
+
 const ignores = ['/node_modules/'];
 
 module.exports = {
   preset: 'react-native',
   testPathIgnorePatterns: [...ignores, 'web.test', 'ssr.test'],
-  collectCoverageFrom: ['./src/**/*.native.{ts,tsx}'],
+  collectCoverageFrom: rebrandedComponents.map((c) => `./src/**/${c}.native.{ts,tsx}`),
```
Comment (chaitanyadeorukhkar) [inline packages/blade/jest.native.config.js:8]:
```suggestion
  collectCoverageFrom: rebrandedComponents.map((component) => `./src/**/${component}.native.{ts,tsx}`),
```

File: packages/blade/jest.web.config.js [hunk 5]
```diff
@@ -12,7 +14,7 @@ const baseConfig = {
   },
   snapshotSerializers: ['<rootDir>/jestStyledComponentsSerializer.js'],
   moduleFileExtensions: ['web.ts', 'web.tsx', 'ts', 'tsx', 'js', 'json', 'node'],
-  testMatch: ['**/*.test.{ts,tsx}'],
+  testMatch: rebrandedComponents.map((c) => `**/${c}.web.test.{ts,tsx}`),
```
Comment (chaitanyadeorukhkar) [inline packages/blade/jest.web.config.js:17]:
```suggestion
  testMatch: rebrandedComponents.map((component) => `**/${component}.web.test.{ts,tsx}`),
```

File: packages/blade/jest.web.config.js [hunk 6]
```diff
@@ -38,15 +40,15 @@ module.exports = {
       ...baseConfig,
       testEnvironment: 'node',
       testPathIgnorePatterns: [...baseConfig.testPathIgnorePatterns, 'web.test'],
-      collectCoverageFrom: ['./src/**/*.ssr.{ts,tsx}'],
-      testMatch: ['**/*.ssr.test.{ts,tsx}'],
+      collectCoverageFrom: rebrandedComponents.map((c) => `./src/**/${c}.ssr.{ts,tsx}`),
```
Comment (chaitanyadeorukhkar) [inline packages/blade/jest.web.config.js:43]:
```suggestion
      collectCoverageFrom: rebrandedComponents.map((component) => `./src/**/${component}.ssr.{ts,tsx}`),
```

File: packages/blade/jest.web.config.js [hunk 7]
```diff
@@ -38,15 +40,15 @@ module.exports = {
...
+      testMatch: rebrandedComponents.map((c) => `**/${c}.ssr.test.{ts,tsx}`),
```
Comment (chaitanyadeorukhkar) [inline packages/blade/jest.web.config.js:44]:
```suggestion
      testMatch: rebrandedComponents.map((component) => `**/${component}.ssr.test.{ts,tsx}`),
```

File: packages/blade/jest.web.config.js [hunk 8]
```diff
@@ -38,15 +40,15 @@ module.exports = {
...
     },
     {
       displayName: 'CSR Test',
       ...baseConfig,
       testEnvironment: 'jsdom',
       testPathIgnorePatterns: [...baseConfig.testPathIgnorePatterns, 'ssr.test'],
-      collectCoverageFrom: ['./src/**/*.web.{ts,tsx}'],
+      collectCoverageFrom: rebrandedComponents.map((c) => `./src/**/${c}.web.{ts,tsx}`),
```
Comment (chaitanyadeorukhkar) [inline packages/blade/jest.web.config.js:51]:
```suggestion
      collectCoverageFrom: rebrandedComponents.map((component) => `./src/**/${component}.web.{ts,tsx}`),
```

File: packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx [hunk 9]
```diff
@@ -16,7 +16,7 @@ const colors: BadgeProps['color'][] = [
 ];
 
 describe('<Badge />', () => {
-  it('should render Badge with default props', () => {
+  it('should render Badge with primary props', () => {
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx:19]: global replace issue?
```suggestion
  it('should render Badge with default props', () => {
```

File: packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx [hunk 10]
```diff
@@ -57,147 +57,141 @@ describe('<Badge />', () => {
     expect(container).toMatchSnapshot();
   });
 
-  it('should render Badge with bold font', () => {
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx:60]: Did we remove bold badge?

File: packages/blade/src/components/BottomSheet/__tests__/BottomSheet.web.test.tsx [hunk 11]
```diff
@@ -234,7 +234,8 @@ describe('<BottomSheet />', () => {
     mockConsoleError.mockRestore();
   });
 
-  it('should compose with Dropdown single select', async () => {
+  // TODO: Dropdown hasn't been rebranded yet thus skipping this test until then
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/BottomSheet/__tests__/BottomSheet.web.test.tsx:237]: Easier to global search later on
```suggestion
  // TODO: Rebranding - Dropdown hasn't been rebranded yet thus skipping this test until then
```

File: packages/blade/src/components/Counter/__tests__/Counter.web.test.tsx [hunk 12]
```diff
@@ -12,7 +12,7 @@ const colors: CounterProps['color'][] = [
 ];
 
 describe('<Counter />', () => {
-  it('should render Counter with default props', () => {
+  it('should render Counter with primary props', () => {
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Counter/__tests__/Counter.web.test.tsx:15]:
```suggestion
  it('should render Counter with default props', () => {
```

File: packages/blade/src/components/Counter/__tests__/Counter.web.test.tsx [hunk 13]
```diff
@@ -38,82 +38,92 @@ describe('<Counter />', () => {
...
-  it('should render low contrast blue variant Counter', () => {
-    const { container } = renderWithTheme(<Counter variant="blue" contrast="low" value={20} />);
+  it('should render subtle emphasis blue color Counter', () => {
+    const { container } = renderWithTheme(<Counter color="primary" emphasis="subtle" value={20} />);
     expect(container).toMatchSnapshot();
   });
 
-  it('should render high contrast blue variant Counter', () => {
-    const { container } = renderWithTheme(<Counter variant="blue" contrast="high" value={20} />);
+  it('should render intense emphasis blue color Counter', () => {
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Counter/__tests__/Counter.web.test.tsx:123]:
```suggestion
  it('should render intense emphasis primary color Counter', () => {
```

File: packages/blade/src/components/Counter/__tests__/Counter.web.test.tsx [hunk 14]
```diff
@@ -38,82 +38,92 @@ describe('<Counter />', () => {
...
+  it('should render intense emphasis neutral color Counter', () => {
+    const { container } = renderWithTheme(
+      <Counter color="neutral" emphasis="intense" value={20} />,
+    );
     expect(container).toMatchSnapshot();
   });
 
-  it('should render low contrast blue variant Counter', () => {
-    const { container } = renderWithTheme(<Counter variant="blue" contrast="low" value={20} />);
+  it('should render subtle emphasis blue color Counter', () => {
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Counter/__tests__/Counter.web.test.tsx:118]:
```suggestion
  it('should render subtle emphasis primary color Counter', () => {
```

File: packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx (hunk 9 above)
Comment (anuraghazra) [reply packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx:19]: fixed

File: packages/blade/jest.web.config.js (hunk 8 above)
Comment (anuraghazra) [reply packages/blade/jest.web.config.js:51]: fixed

File: packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx (hunk 10 above)
Comment (anuraghazra) [reply packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx:60]: Yes.

File: packages/blade/src/components/BottomSheet/__tests__/BottomSheet.web.test.tsx (hunk 11 above)
Comment (anuraghazra) [reply packages/blade/src/components/BottomSheet/__tests__/BottomSheet.web.test.tsx:237]: done.

File: packages/blade/src/components/ProgressBar/__tests__/ProgressBar.native.test.tsx [hunk 15]
```diff
@@ -47,67 +49,11 @@ describe('<ProgressBar />', () => {
...
-
-  it('should render low contrast neutral intent ProgressBar', () => {
-    const { toJSON } = renderWithTheme(<ProgressBar intent="neutral" contrast="low" value={20} />);
-    expect(toJSON()).toMatchSnapshot();
-  });
-
-  it('should render high contrast neutral intent ProgressBar', () => {
-    const { toJSON } = renderWithTheme(<ProgressBar intent="neutral" contrast="high" value={20} />);
-    expect(toJSON()).toMatchSnapshot();
+  colors.forEach((color) => {
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/ProgressBar/__tests__/ProgressBar.native.test.tsx:52]: 👏

File: packages/blade/src/components/Tag/Tag.stories.tsx [hunk 16]
```diff
@@ -100,19 +98,19 @@ Disabled.args = {
...
-  children,
-  onSubmit,
-}: {
-  children: React.ReactElement[];
-  onSubmit: DOMAttributes<HTMLFormElement>['onSubmit'];
-}): React.ReactElement => {
-  if (isReactNative()) {
-    return <Box>{children}</Box>;
-  }
+// const CrossPlatformForm = ({
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Tag/Tag.stories.tsx:101]: Why did you remove this?

File: packages/blade/src/components/Typography/Text/__tests__/Text.web.test.tsx [hunk 17]
```diff
@@ -70,7 +86,7 @@ describe('<Text />', () => {
     expect(() =>
       renderWithTheme(
         // @ts-expect-error testing failure case when size='small' is passed with variant='caption'
-        <Text type="normal" variant="caption" truncateAfterLines={3} size="small">
+        <Text color="normal" variant="caption" truncateAfterLines={3} size="small">
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Typography/Text/__tests__/Text.web.test.tsx:89]: Color normal is not valid, right? We're testing for incorrect size & not incorrect color here

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Review by kamleshchandnani — APPROVED** [review]
